### PR TITLE
Inital models

### PR DESCRIPTION
**Pull request by asifch**
[model.txt](https://github.com/sparkymat/SupplyTracker/files/2300585/model.txt)

### CODE REVIEW

File: db/migrate/20180819185350_add_fields_to_relief_camp.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class AddFieldsToReliefCamp < ActiveRecord::Migration[5.2]
+  def change
+    add_column :relief_camps, :latitude, :float
+    add_column :relief_camps, :longitude, :float
```
Comment (cibingeorge) [inline db/migrate/20180819185350_add_fields_to_relief_camp.rb]: longitude double precision,
    latitude double precision,

File: db/migrate/20180819185350_add_fields_to_relief_camp.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    add_column :relief_camps, :address, :text
+    add_column :relief_camps, :phone_no, :string
```
Comment (cibingeorge) [inline db/migrate/20180819185350_add_fields_to_relief_camp.rb]: convert strings to text? use model validations to manage constraints?

File: db/migrate/20180819185350_add_fields_to_relief_camp.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    add_column :relief_camps, :whatsapp_no, :string
+    add_column :relief_camps, :email, :string
+    add_column :relief_camps, :contact_person, :string
+    add_column :relief_camps, :coordinator_id, :sting
```
Comment (cibingeorge) [inline db/migrate/20180819185350_add_fields_to_relief_camp.rb]: typo here.. should be string

File: db/migrate/20180819185350_add_fields_to_relief_camp.rb (hunk 1 above)
Comment (sparkymat) [reply db/migrate/20180819185350_add_fields_to_relief_camp.rb]: fixing in my copy of this PR

File: db/migrate/20180819185350_add_fields_to_relief_camp.rb (hunk 2 above)
Comment (sparkymat) [reply db/migrate/20180819185350_add_fields_to_relief_camp.rb]: postgresql driver automatically puts it as varchar without a limit. should be alright.

File: db/migrate/20180819185350_add_fields_to_relief_camp.rb (hunk 3 above)
Comment (sparkymat) [reply db/migrate/20180819185350_add_fields_to_relief_camp.rb]: it already has a `coordinator_id`

Comment (cibingeorge) [reply db/migrate/20180819185350_add_fields_to_relief_camp.rb]: @sparkymat the type should be string..it says sting
```
:st  r  ing 
     ^
     |
```

**Pull request merged**